site/_posts/2021-10-26-how-to-set-up-single-sign-on-between-fusionauth-wordpress.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog-post
+title: How to Set Up Single Sign-On Between FusionAuth and WordPress
+description: In this tutorial, you'll be learning how to implement OAuth SSO for Joomla users.
+author: Aniket Bhattacharyea
+image: blogs/wordpress-sso-fusionauth/how-to-set-up-single-sign-on-sso-between-fusionauth-and-wordpress-header-image.png
+category: blog
+tags: tutorial tutorial-php client-php tutorial-integration
+excerpt_separator: "<!--more-->"
+---
+
+User authentication is one of the most important parts of building any application, and almost every application features some sort of authentication. However, it quickly gets difficult to manage when you have multiple related but independent applications to maintain.
+
+<!--more-->
+
+Let's say you're building a photo-sharing application that lets a user sign up with a username and password to share their pictures. Your team might later decide to [create a forum using WordPress](https://wordpress.com/go/tutorials/create-a-forum-with-wordpress-com/) to help support your app, but this new project isn't tied to your original application. It uses an entirely different tech stack, database, and hosting setup.
+
+This means your users need to sign up _again_ on this second website and remember another set of usernames and passwords. Giving users two entirely different sets of credentials is not ideal, so that's where Single Sign-On comes in.
+
+Single Sign-On, commonly known as SSO, is an authentication mechanism that enables users to use a single set of credentials to log in to multiple different applications or websites. In a typical SSO setup, you have a service provider—the application the user is trying to log in to—and an identity provider like [FusionAuth](https://fusionauth.io), which handles the authentication. This means your users need only one set of credentials to log into all of your applications.
```

Comment:
I'd say "log in to", or "log into". In this paragraph, you use both interchangeably. 

Here's a forum where people hotly debate which one is better.  It looks like both are ok. 
https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to